src/components/Navbar/index.tsx

```diff
@@ -57,21 +57,11 @@ const Navbar: React.FC = () => {
                         )}
                     </li>
                     <ul className={`${menuOpen ? 'block' : 'hidden'} absolute top-[8vh] right-0 bg-gray-800 p-2 z-10`}>
-                        <li>
-                            <a href="#" className="text-white hover:bg-gray-700 block px-4 py-2">
-                                Profile
-                            </a>
-                        </li>
                         <li>
                             <a href="#" className="text-white hover:bg-gray-700 block px-4 py-2">
                                 Dashboard
                             </a>
                         </li>
-                        <li>
-                            <a href="#" className="text-white hover:bg-gray-700 block px-4 py-2">
-                                Settings
-                            </a>
-                        </li>
                         <li>
                             <a href={TINY_API_LOGOUT} className="text-white hover:bg-gray-700 block px-4 py-2">
```

Comment:
make an array of nav items and map it, If it is repetitive. This will make nav items dynamic and scalable

Reply:
I don't think using this approach is the best option for only two elements.

Reply:
There is a minor performance overhead when using `map` for a small number of items